wd.html

```diff
@@ -177,21 +177,20 @@
 
 <h3>Firefox</h3>
 
-<p>The driver for Firefox is maintained by the Selenium project
- and works by installing a browser add-on
- in Firefox' profile before starting the browser.
- The WebDriver client then communicates with that add-on
- to instruct the browser what to do.
+<p>Starting with Selenium 3, Mozilla has taken over implementation of Firefox Driver, with <a href=https://github.com/mozilla/geckodriver/releases>Geckodriver</a>, which must be added to your PATH.
```

Comment:
* “implementation of Firefox Driver” → “the driver implementation for Firefox”
* Drop `/releases` from URL
* Drop “, which must be added to your PATH”; this is explained later and is not strictly necessary